apps/site/assets/ts/stop/components/DepartureTimes.tsx

```diff
@@ -228,16 +228,16 @@ const departureTimeRow = (
           {formattedTimes.map((time, index) => {
             const classes = departureTimeClasses(time, index);
             return (
-              <>
+              <div
+                key={`${time.displayString}-departure-times`}
```

Comment:
Good catch!
question: Are the display times guaranteed to be unique? May be worth appending [useId](https://react.dev/reference/react/useId) to be sure. Maybe two buses could be predicted to arrive at the same time when there is bunching going on

Reply:
Great Question!
I didn't think about bus bunching.  Yeah this will fail if 2 buses are predicted to arrive in the same minute.  It looks like [useId should not be used to generate key ids](https://react.dev/reference/react/useId#:~:text=useId%20should%20not%20be%20used%20to%20generate%20keys%20in%20a%20list.%20Keys%20should%20be%20generated%20from%20your%20data.).  This might need some more thoughts about it.